extensions/common/context.cc

```diff
@@ -156,7 +156,10 @@ google::protobuf::util::Status extractLocalNodeMetadata(
   return extractNodeMetadata(node, node_info);
 }
 
-void populateHTTPRequestInfo(bool outbound, RequestInfo* request_info) {
+// Host header is used if use_host_header_fallback==true.
```

Comment:
Host header will be used only when the destination service is missing, so this comment is misleading.

Reply:
Updated comment.

Reply:
oop,s @PiotrSikora I updated comment only in the other PR.